pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -309,3 +309,8 @@ def teardown(self) -> None:
     @classmethod
     def register_plugins(cls, plugin_registry):
         pass
+
+    @property
+    def should_save_checkpoint(self) -> bool:
+        """Returns whether the checkpoint should be saved (rank based)"""
+        return self.is_global_zero
```

Comment:
i think this should mention the rank in the property name, like `should_rank_save_checkpoint`
otherwise it can be confused with the checkpoint saving logic in the model checkpoint callback

Reply:
Agreed. Updated!